felayout_t3kit/dev/styles/main/contentElements/heroImage.less

```diff
@@ -86,6 +105,7 @@
     transition: none;
 }
 
+.top-content .slider-container-effect,
```

Comment:
Not sure that `top content` here in the right place. HeroImage element can be not only inside `top content`. So, use CSS styles which depend on `top content` wrapper - wrong decision.